src/schemas/json/dependabot-2.0.json

```diff
@@ -663,6 +663,7 @@
         "npm",
         "nuget",
         "pip",
+        "pip-compile",
```

Comment:
[The documentation](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem) shows that while `pip-compile` is a supported _tool_, the _YAML value_ is still just `pip`.

> ![image](https://github.com/user-attachments/assets/62124b93-5578-4bb9-bcea-f0d83468ba9f)


I'll submit a PR to revert this.